### PR TITLE
add date validation for journey

### PR DESCRIPTION
**Pull request by joanico**
User la bele hatama data iha iha tempu oin mai so hatama deit data iha tempu uluk no agora.

### CODE REVIEW

File: map/templates/map/viajen_form.html [hunk 1]
```diff
@@ -7,8 +7,9 @@
 {% load i18n %}
 
     <!-- Fengyuan Chen's Datepicker -->
-    <link rel="stylesheet" type="text/css" href="{% static 'css/datepicker.min.css' %}">
-    <script type="text/javascript" src="{% static 'js/datepicker.min.js' %}"></script>
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```
Comment (PeteCoward) [inline map/templates/map/viajen_form.html:10]: you are including jquery ui css from a cdn

File: map/templates/map/viajen_form.html [hunk 2]
```diff
@@ -7,8 +7,9 @@
...
+    <script src="{% static 'js/jquery-1.12.4.js' %}"></script>
+    <script src="{% static 'js/jquery-ui.js' %}"></script>
```
Comment (PeteCoward) [inline map/templates/map/viajen_form.html:12]: here you are including jquery ui js from the source tree.

It is better to either deliver all 3rd party assets from a CDN, or copy them ALL into your source code

File: map/templates/map/viajen_form.html [hunk 3]
```diff
@@ -7,8 +7,9 @@
 {% load i18n %}
 
     <!-- Fengyuan Chen's Datepicker -->
```
Comment (PeteCoward) [inline map/templates/map/viajen_form.html:9]: remove this

File: map/templates/map/viajen_form.html [hunk 4]
```diff
@@ -7,8 +7,9 @@
...
-    <link rel="stylesheet" type="text/css" href="{% static 'css/datepicker.min.css' %}">
-    <script type="text/javascript" src="{% static 'js/datepicker.min.js' %}"></script>
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <script src="{% static 'js/jquery-1.12.4.js' %}"></script>
```
Comment (PeteCoward) [inline map/templates/map/viajen_form.html:11]: definitely remove this

File: map/static/css/timor.css [hunk 5]
```diff
@@ -95,6 +95,10 @@ input#id_date_1 {
     margin-bottom: 10px;
 }
 
+.ui-datepicker .ui-datepicker-prev span, .ui-datepicker .ui-datepicker-next span {
```
Comment (eraulo) [inline map/static/css/timor.css:98]: @joanico Why we need double timor.css in here?

File: map/templates/map/viajen_form.html [hunk 6]
```diff
@@ -6,9 +6,9 @@
 {% block content %}
 {% load i18n %}
 
-    <!-- Fengyuan Chen's Datepicker -->
-    <link rel="stylesheet" type="text/css" href="{% static 'css/datepicker.min.css' %}">
-    <script type="text/javascript" src="{% static 'js/datepicker.min.js' %}"></script>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/jquery-ui.css' %}">
+    <link rel="stylesheet" href="{% static 'css/timor.css' %}">
```
Comment (eraulo) [inline map/templates/map/viajen_form.html:10]: @joanico timor.css already exited in the base.html

File: map/templates/map/viajen_form.html [hunk 7]
```diff
@@ -6,9 +6,9 @@
...
+    <script src="{% static 'js/jquery-ui.js' %}"></script>
```
Comment (eraulo) [inline map/templates/map/viajen_form.html:11]: @joanico why need double jquery?

File: map/static/css/timor.css (hunk 5 above)
Comment (joanico) [reply map/static/css/timor.css:98]: It's for the modified datepicker display the button to  previous and next.

File: map/templates/map/viajen_form.html (hunk 7 above)
Comment (joanico) [reply map/templates/map/viajen_form.html:11]: It's not double jquery it's for jquery validation datepicker.

Comment (joanico) [reply map/templates/map/viajen_form.html:11]: It's changed datepicker to datepicker validation with jquery

**Pull request merged**